pages/whats-new/overview.md

```diff
@@ -21,37 +15,123 @@ redirect_from:
 ---
-
+<!-- Combine posts and short posts into a single feed -->
 {% assign all_posts = site.posts | concat: site.posts_short | sort: "date" | reverse %}
 
-{% include site-card-list.html
-  cards=page.cards
-  listClasses="margin-top-6"
-  listItemClasses="desktop:grid-col-6"
-%}
+<!--
+  Set product release card variables
+  - Set card list classes
+  - Set card strings
+-->
+{% assign product_grid_list_classes = "usa-card site-component-card desktop-lg:grid-col-4" %}
+{% capture uswds_link_text %} USWDS {{ site.uswds_version }}{%- endcapture %}
+{% capture uswds_compile_link_text %} USWDS Compile {{ site.uswds_compile_version }}{%- endcapture %}
+{% capture uswds_design_kit_link_text %} USWDS design kit {{ site.uswds_design_kit_version }}{%- endcapture %}
+
+<!--
+  Set monthly call card variables
+  - Get data from the most recent monthly call
+  - Set card strings
+-->
+{% assign monthly_call = site.data.monthly-calls.videos[0]  %}
+{% capture monthly_call_heading %} {{ monthly_call.date }}: {{monthly_call.title}} {%- endcapture %}
+{% capture monthly_call_image %} {{site.baseurl }}/img/uswds-logo/4c-lg-on-black.svg {%- endcapture %}
+{% capture monthly_call_url %} {{ site.baseurl }}/about/monthly-calls {%- endcapture %}
+
+{:.whats-new__heading.margin-top-4}
+On this page
+
+{% include jump-links.html %}
 
-{:.margin-top-2.text-normal.font-lang-md.text-gray-70}
-## News and updates
+<div class="bg-gray-5 padding-3 margin-top-4">
+  <h2 class="whats-new__heading margin-top-0" id="latest-releases">Latest releases</h2>
+  <ul class="usa-card-group whats-new-card-group">
+    <li class="{{ product_grid_list_classes }}">
+      {% include site-card.html
+        subheading="Design system"
+        heading="USWDS"
+        body="The official U.S. design system, enabling the government to build fast, accessible, mobile-friendly websites."
+        linkText=uswds_link_text
+        linkUrl="https://github.com/uswds/uswds/releases"
+        subheadingElement="span"
+        linkClasses="padding-x-105"
+      %}
+    </li>
+    <li class="{{ product_grid_list_classes }}">
+      {% include site-card.html
+        subheading="Compilation tool"
+        heading="USWDS Compile"
+        body="A tool that makes it easy to customize and compile USWDS Sass into browser-readable CSS."
+        linkText=uswds_compile_link_text
+        linkUrl="https://github.com/uswds/uswds-compile/releases"
+        subheadingElement="span"
+        linkClasses="padding-x-105"
+      %}
+    </li>
+    <li class="{{ product_grid_list_classes }}">
+      {% include site-card.html
+        subheading="Design assets"
+        heading="USWDS for designers"
+        body="The official USWDS design kits in Sketch and Figma."
+        linkText=uswds_design_kit_link_text
+        linkUrl="https://www.design_kit.com/community/file/1440921849343185329"
+        subheadingElement="span"
+        linkClasses="padding-x-105"
+      %}
+    </li>
+  </ul>
+</div>
```

Comment:
> [!note]
> I could not use site-card-list.html here because I needed to pull site data for this card content

Reply:
Confirming it's not possible to pass data into front matter in Jekyl without plugins or potentially hacky work arounds. This is something I've struggled with in the past.

Reply:
I see includes that use site data.

https://github.com/search?q=repo:uswds/uswds-site+site.+language:HTML+path:/%5E_includes%5C//&type=code